acme/acme/client.py

```diff
@@ -599,6 +600,8 @@ def _check_response(cls, response, content_type=None):
         return response
 
     def _send_request(self, method, url, *args, **kwargs):
+    #pylint: disable=too-many-locals
```

Comment:
Please line this comment up with the beginning of the block quote, and remove the extra line below it.